exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -54,6 +59,13 @@
   static final Set<MetricDataType> DOUBLE_TYPES = ImmutableSet.of(DOUBLE_GAUGE, DOUBLE_SUM);
   private static final int MIN_TIMESTAMP_INTERVAL_NANOS = 1000000;
 
+  // Mapping outlined at https://cloud.google.com/monitoring/api/resources#tag_gce_instance
+  private static final Map<String, AttributeKey<String>> gceMap =
+      Map.of(
```

Comment:
This method was added in Java 9, and I believe we're aiming for artifacts to be compatible with Java 8.

(Ideally we should verify this in CI instead of burdening developers with a manual check.)